### PR TITLE
Antora 3.0 related updates

### PR DESCRIPTION
**Pull request by travier**
See https://pagure.io/fedora-docs/docs-fp-o/pull-request/178

### CODE REVIEW

**Comment by travier** [comment]
CC @cverna

File: preview.sh [hunk 1]
```diff
@@ -1,41 +1,18 @@
...
-	    fi
-
-    else
-        echo ""
-	    echo "Error: Container runtime haven't been found on your system. Fix it by:"
-	    echo "$ sudo dnf install podman"
-	    exit 1
-    fi
-
+    docker run --rm -v "$(pwd):/antora:ro" -v "$(pwd)/nginx.conf:/etc/nginx/conf.d/default.conf:ro" -p 8080:80 nginx
```
Comment (LorbusChris) [inline preview.sh:8]: Maybe we should take this as an opportunity to suggest using podman on MacOS now?

Comment (rhatdan) [reply preview.sh:8]: SGTM

**Comment by travier** [comment]
For now this is just an update to match what's in https://pagure.io/fedora-docs/template and to fix issues with the Antora 3 update.

I agree that we should recommend podman on macOS but this is a different change and would need to be done for the template too.

**Review by cverna — APPROVED** [review]

**Pull request merged**